Source/DOM classes/SVG-DOM/SVGRect.h

```diff
@@ -38,3 +38,11 @@ CGRect CGRectFromSVGRect( SVGRect rect );
 
 /** Convenience method to convert to ObjectiveC's kind of size - ONLY the width and height of this rect */
 CGSize CGSizeFromSVGRect( SVGRect rect );
+
+NSString * _Nonnull NSStringFromSVGRect( SVGRect rect );
+
+#if SVGKIT_MAC
```

Comment:
Maybe these portable method should be only used privatelly, but not in public API.

Reply:
An alternative is to use preprocessor macros to redefine, i.e.  NSStringFromCGRect to NSStringFromRect:

````
#define NSStringFromCGRect NSStringFromRect
````

Although I agree that it could be put in a better place.